app/components/time_based_activity_heatmap/time_based_activity_heatmap.css

```diff
@@ -0,0 +1,4 @@
+.TimeBasedActivityHeatmap {
+  height: 37rem;
```

Comment:
What do you mean? Like why not set a height for the section, which is the `div` with class `TimeBasedActivityHeatmap`'s parent?

I want to set the height of the `div` that will render the heatmap because I want to avoid the screen jumping when the JS is rendered and the div changes from a height of `0` to `37rem`. The `div` parent has other children, like the heading so I don't want to give it a fixed height.

Not sure if that was your question.

Reply:
jep, makes sense, thanks